lib/spotlight/dor/indexer.rb

```diff
@@ -48,47 +47,52 @@ def solr_document resource
     def resource druid
       Harvestdor::Indexer::Resource.new harvestdor, druid
     end
-    
+
     private
     # def add_something_to_solr_doc sdb, solr_doc
-    #   solr_doc['some_field_name_tsi'] = 'some value'
+    #   solr_doc['some_field_name_tsi'] = 'some value'   # OR
+    #   insert_field solr_doc,'some_multivalued_field_name_ssim',array_of_values
     # end
 
+    def add_donor_tags sdb,solr_doc
+      donor_tags = sdb.smods_rec.note.select { |n| n.displayLabel == 'Donor tags' }.map(&:content)
+      insert_field solr_doc, 'donor_tags', donor_tags, :stored_searchable
```

Comment:
I think if they are to be a facet, they should not be "text" because i *think* the facet will show terms rather than the full string.  I think there may be a :facetable thingy?   I believe they would be searched via a catch-all text field???  This *may* be the relevant link:

https://github.com/projecthydra/solrizer/blob/05e6a01b32553b9df434eacc35c0888f8bbc8050/lib/solrizer/default_descriptors.rb